backend/molgenis-emx2-beacon-v2/src/main/java/org/molgenis/emx2/beaconv2/endpoints/EntryTypes.java

```diff
@@ -11,20 +11,20 @@
 import org.molgenis.emx2.beaconv2.BeaconSpec;
 import org.molgenis.emx2.beaconv2.EntryType;
 import spark.Request;
```

Comment:
principieel zou je liefst de Request/Response niet nodig hebben hier. Want nu heb je onnodige dependency. Liefst pass je wat je nodig hebt. Kan dat? Of is dat teveel nu (je zit met oude code en dus even zo houden dan maar?)